pasteit/views/index.ejs

```diff
@@ -52,90 +51,18 @@
-                                <li class="help-title">When Viewing A Paste</li>
-                                <li>
-                                    <ul>
-                                        <li class="help-line">
-                                            <span class="hotkey">ALT+G</span>
-                                            Go to a line
-                                        </li>
-                                        <li class="help-line">
-                                            <span class="hotkey">ALT+CTRL+E</span>
-                                            Edit the paste
-                                        </li>
-                                        <li class="help-line">
-                                            <span class="hotkey">ALT+R</span>
-                                            Show the raw code
-                                        </li>
-                                    </ul>
-                                </li>
-                            </ul>
-                        </li>
-                        <li class="help-subject">
-                            There are URL options!
-                            <ul>
-                                <li class="help-title">
-                                    When Creating A Paste
-                                </li>
-                                <li>
-                                    <ul>
-                                        <li class="help-line">
-                                            ?lang=Javascript to default to javascript
-                                        </li>
-                                    </ul>
-                                </li>
-                                <li class="help-title">When Viewing A Paste</li>
-                                <li>
-                                    <ul>
-                                        <li class="help-line">#L-N Jump to line number N</li>
-                                    </ul>
-                                </li>
-                            </ul>
-                        </li>
-                    </ul>
-                </div>
-            </div>
-            <span id="help-bubble">?</span>
-        </div>
-        <script type="text/javascript">//<![CDATA[
-var _gaq = _gaq || [];
-_gaq.push(['_setAccount', 'UA-28074244-1']);
-_gaq.push(['_trackPageview']);
+        <script type="text/javascript">
+        //<![CDATA
+            var _gaq = _gaq || [];
+            _gaq.push(['_setAccount', 'UA-28074244-1']);
+            _gaq.push(['_trackPageview']);
 
-(function() {
-    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-})();
-//]]>
+            (function() {
```

Comment:
Is there a privacy concern here? We want to keep site usage anonymous due to the legal problems that occur for us when people post illicit material. I think google analytics will violate that.

Reply:
Yeah, I guess it's an artifact of the original refheap.